nucleus/cluster/admin/src/main/manpages/com/sun/enterprise/v3/admin/cluster/start-cluster.1

```diff
@@ -38,6 +39,23 @@ DESCRIPTION
 OPTIONS
        --help, -?
            Displays the help text for the subcommand.
+       
+       --debug
+           Specifies whether each instance in the cluster is started with Java Platform
+           Debugger Architecture (JPDA)
+
+           (http://www.oracle.com/technetwork/java/javase/tech/jpda-141715.html)
```

Comment:
Link are alive?

Reply:
Probably some updating needed, but across the board.

This one seems to be the correct one: https://docs.oracle.com/en/java/javase/17/docs/specs/jpda/jpda.html